api/base/serializers.py

```diff
@@ -249,7 +249,8 @@ def to_internal_value(self, data):
             type_ = self.root.Meta.type_
 
         if type_ != data:
-            raise Conflict()
+            raise Conflict(detail=('The type you specified in the URL "' +
```

Comment:
This doesn't match what I suggested. Could it not be implemented that way or was it incorrect?
![screen shot 2016-06-02 at 3 56 20 pm](https://cloud.githubusercontent.com/assets/6599111/15758938/c5b911fc-28da-11e6-8a2a-0fd84db6d9ee.png)